Codecademy/gamut
# fix(ScoreSummary): Use List role for subjects in ScoreSummary

### Overview


- Refactor ScoreSummary to add a list role to subject scores



### PR Checklist

- [x] Related to JIRA ticket: [EGG-3032]

#### Testing Instructions

Test in the ScoreSummary page on storybook, and/or in the PR environments in the monolith and portal. In the monolith, you can find the ScoreSummary component on the completion screen after finishing a cumulative assessment ([example assessment](https://pr-34399-monolith.dev-eks.codecademy.com/cumulative-assessment/journeys/full-stack-engineer/paths/fscj-22-building-interactive-websites/parts/1) - as an admin you can skip to the end of the assessment by using the "Jump to last question" option in the settings menu in the header), and in portal you can find it in the [career journey curriculum page](https://tengu.codecademy.com/career-journey/full-stack-engineer?PR_ENV=portal-app-pr-1106), in the expanded exam section below a path for any completed exam.

Navigate through the ScoreSummary component using VoiceOver and ensure that the subjects are announced as a list, including both tested and untested subjects.

#### PR Links and Envs

| Repository   | PR Link                                                  | PR Env                                                   |
| :----------- | :------------------------------------------------------- | :------------------------------------------------------- |
| Monolith     | [Monolith PR](https://github.com/codecademy-engineering/Codecademy/pull/34399)  | [Monolith Env](https://pr-34399-monolith.dev-eks.codecademy.com/) |
| Portal       | [Portal Link](https://github.com/codecademy-engineering/mono/pull/1106)  | [Portal Env](https://pr-34399-monolith.dev-eks.codecademy.com/)   |




[EGG-3032]: https://codecademy.atlassian.net/browse/EGG-3032?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
